lib/src/authentication/create/create_account_view.dart

```diff
@@ -0,0 +1,107 @@
+  Widget build(BuildContext context) {
+    final AppLocalizations l10n = AppLocalizations.of(context);
+
+    final Widget textDivider = Expanded(
+      child: Padding(
+        padding: const EdgeInsets.only(left: 20.0, right: 20.0),
+        child: Divider(
+          color: Theme.of(context).colorScheme.primaryContainer,
+        ),
+      ),
+    );
+
+    return ViewModelBuilder<CreateAccountViewModel>.reactive(
+      viewModelBuilder: () => CreateAccountViewModel(GoRouter.of(context)),
+      builder: (context, viewModel, child) => AppScaffold(
+        appBar: AppBar(
+          backgroundColor: Colors.transparent,
+          leading: IconButton(
+            key: const Key("create_account_view_return"),
+            icon: const Icon(Icons.arrow_back_rounded),
+            onPressed: () => context.pop(),
+          ),
+        ),
+        body: Form(
+          key: viewModel.formKey,
+          child: Padding(
+            padding: const EdgeInsets.only(top: 8.0),
+            child: Column(
+              mainAxisAlignment: MainAxisAlignment.center,
+              children: [
+                InputEmail(controller: viewModel.emailController),
+                InputPassword(controller: viewModel.passwordController),
+                InputPassword(
+                  controller: viewModel.passwordConfirmationController,
+                  passwordToConfirmController: viewModel.passwordController,
+                ),
+                SizedBox(
+                  width: double.infinity,
+                  child: Padding(
+                    padding: const EdgeInsets.all(12.0),
+                    child: FilledButton(
+                      key: const Key(
+                          "create_account_view_create_account_button"),
+                      onPressed: viewModel.createAccount,
+                      child: Text(
+                        l10n.create_account_view_create_account,
+                        style: const TextStyle(fontSize: 20.0),
+                      ),
+                    ),
+                  ),
+                ),
+                Padding(
+                  padding: const EdgeInsets.only(top: 28.0, bottom: 28.0),
+                  child: Row(
+                    children: [
+                      textDivider,
+                      Text(
+                        l10n.create_account_view_or_separator,
+                        style: const TextStyle(fontSize: 20.0),
+                      ),
+                      textDivider
+                    ],
+                  ),
+                ),
+                Row(
+                  children: [
+                    Expanded(
+                      child: ButtonLogo.google(
+                        onPressedFunction: viewModel.createAccountGoogle,
+                      ),
+                    ),
+                    // Display the Apple button only on iOS
+                    if (defaultTargetPlatform == TargetPlatform.iOS)
+                      Expanded(
```

Comment:
Avoid using `Expanded` but `Row.mainAlignment: MainAxisAlignment.center` instead (see [Sign in view l.131](https://github.com/RoadTripMoustache/kana_to_kanji/blob/337ce15ddbd8e14f66e56426c3d8a6dd55611a2c/lib/src/authentication/sign_in/sign_in_view.dart#L131))

Reply:
Updated